system-addon/lib/SectionsManager.jsm

```diff
@@ -20,10 +23,10 @@ const BUILT_IN_SECTIONS = {
     id: "topstories",
     pref: {
       titleString: {id: "header_recommended_by", values: {provider: options.provider_name}},
-      descString: {id: options.provider_description || "pocket_description"},
+      descString: {id: ABOUT_PREFERENCES ? "prefs_topstories_description" : options.provider_description || "pocket_description"},
```

Comment:
I suppose this condition can be removed because we're no longer experimenting. Separately, should we still just leave about preferences as a feed that someone could disable.. and somewhat break things? I guess that user would just not see activity stream prefs in about:preferences…